src/screens/EventDetailsScreen/component.js

```diff
@@ -0,0 +1,225 @@
+// @flow
+import React from "react";
+import { View, StyleSheet, ScrollView } from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+import formatDate from "date-fns/format";
+import isSameDay from "date-fns/is_same_day";
+import Header from "./Header";
+import IconItem from "./IconItem";
+import CategoryLabel from "./CategoryLabel";
+import EventMap from "./EventMap";
+import Text from "../../components/Text";
+import Button from "../../components/Button";
+import {
+  eventDetailsBgColor,
+  eventDetailsHeaderBgColor
+} from "../../constants/colors";
+import text from "../../constants/text";
+import strings from "../../constants/strings";
+import type { Event } from "../../integrations/cms";
+
+const locale = "en-GB";
+
+type Props = {
+  navigation: NavigationScreenProp<{ params: { eventId: String } }>,
+  event: Event
+};
+
+const renderEventOverview = event => {
+  const startTime = new Date(event.fields.startTime[locale]);
+  const endTime = new Date(event.fields.endTime[locale]);
+  const dateFormat = "DD MMMM YYYY";
+  const timeFormat = "HH:mm";
+  const dateDisplay = isSameDay(startTime, endTime)
+    ? formatDate(startTime, dateFormat)
+    : `${formatDate(startTime, dateFormat)} - ${formatDate(
+        endTime,
+        dateFormat
+      )}`;
+  const timeDisplay = `${formatDate(startTime, timeFormat)} - ${formatDate(
+    endTime,
+    timeFormat
+  )}`;
+
+  return (
+    <View style={styles.content}>
+      <Text type="h1">{event.fields.name[locale]}</Text>
+      <View style={styles.categoryLabelContainer}>
+        {event.fields.eventCategories[locale].map(categoryName => (
+          <CategoryLabel key={categoryName} categoryName={categoryName} />
+        ))}
+      </View>
+      <View style={styles.iconItemWrapper}>
+        <IconItem icon={<Text type="xSmall">icn</Text>} title={dateDisplay}>
```

Comment:
The usage of the `<IconItem>` is a bit confusing. I did not expect the `children` to be rendered below the `title`. I think one of these would be easier to understand:

```jsx
<IconItem icon={<Text type="xSmall">icn</Text>}>
  <Text>{dateDisplay}</Text>
  <Text type="small">{timeDisplay}</Text>
</IconItem>
```

```jsx
<IconItem
  icon={<Text type="xSmall">icn</Text>}
  title={dateDisplay}
  content={timeDisplay}
/>
```

Reply:
I think the second option is probably clearer. You're right I think I was trying to be too clever with the `children` node. I'll change that now...

Reply:
Done

Reply:
I wonder if `content` could even default to `<Text type="small">...</Text>`. 🤔